ValidatorPanel.php

```diff
@@ -769,6 +769,11 @@ class ValidatorPanel implements Tracy\IBarPanel
 	 * @var array|\LibXMLError[]
 	 */
 	private $errors = array();
+	
+	/**
+	 * @var array
+	 */
+	private $disableValidationInPresenters = array();
```

Comment:
`$presentersToDisableValidationFor` sounds better in 🇬🇧

Reply:
Yes it sound better i changed this name